components/Feature/VulnerabilitiesGrid/ResourceCard/index.js

```diff
@@ -32,9 +32,9 @@ const ResourceCard = ({
   const trimLength = (s, length) => s.length > length ? s.substring(0, length) + "..." : s
 
   const selfReferralElement = (selfReferral == 'No') ? 'Referral required' : 'Self referral'
-  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<a href={websites[0]} target="_blank" rel="noopener noreferrer">{websites[0]}</a>))
+  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<><a href={website} target="_blank" rel="noopener noreferrer">{website}</a><br/></>))
```

Comment:
that tag didn't look quite right, I think it should be something like this
```suggestion
  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<a href={website} target="_blank" rel="noopener noreferrer">{website}</a><br/></>))
```

Reply:
I've just put it inside a div and removed fragment and `</br>` 😅